osu.Game/Rulesets/Scoring/ScoreProcessor.cs

```diff
@@ -314,11 +314,11 @@ protected virtual void ApplyResult(JudgementResult result)
 
             JudgedHits++;
 
-            if (result.Type != HitResult.None)
-                scoreResultCounts[result.Type] = scoreResultCounts.GetOrDefault(result.Type) + 1;
-
             if (result.Judgement.AffectsCombo)
             {
+                if (result.Type != HitResult.None)
```

Comment:
Moving this in here in this PR is very out of place. I'm not sure if this is correct or not, but it does not fall under the title of this PR and has no further information on what you are trying to fix (or where you see this being an issue).

Please PR separately with full information.

Reply:
Gonna hold off PRing the split out change until I finish a test that helps us verify autoplay gets as many greats as maxcombo